src/style.css

```diff
@@ -1 +1,96 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+  font-style: normal;
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "Roboto";
+  src: url("https://fonts.googleapis.com/css2family=Roboto:wght@300;700&display=swap");
+  font-style: bold;
+  font-weight: 700;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  margin: 0;
+  --main-color: #00acdc;
+}
+
+.container {
+  padding: 0 50px;
+
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: relative;
+  width: 40px;
+  height: 40px;
+  background-image: url("./images/logo.png");
+  background-repeat: no-repeat;
+}
+
+.nav {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  font-size: 12px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  text-transform: uppercase;
+  padding-left: 0;
+  margin: 0;
+}
+
+.nav__list:first-child::after {
+  content: "";
+  position: absolute;
+  background: var(--main-color);
+  height: 4px;
+  width: 37px;
+  margin: 56px auto 0;
+  border-radius: 8px;
+}
```

Comment:
move it to `.is-active::after`

Reply:
If they change like this, then the tests will not pass. Tests do not see the underlined line

Reply:
I solved the problem by inserting the is-active class twice:

<ul class="nav__list is-active">
            <a class="box is-active" href="#apple">Apple</a>
          </ul>

Reply:
I solved the problem by inserting the is-active class twice in *.html